signals/headers/headers_no_recipients_in_org_domain.yml

```diff
@@ -1,4 +1,5 @@
-name: "Headers: No recpients in $org_domains list"
```

Comment:
FYI this needs to be changed in both Mantis and Hydra (we were matching the typo before)

Reply:
yep... I'll pick it up during model training, and then PR an updated config in Hydra